articles/azure/azs-how-add-data-managed-portal.md

```diff
@@ -0,0 +1,113 @@
+---
+title: How to add a managed disk to a virtual machine using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help adding managed disks to a virtual machine on UKCloud for Microsoft Azure Stack
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Add a managed disk to a virtual machine
+toc_sub3:
+toc_sub4:
+toc_title: Add a managed disk to a virtual machine - portal
+toc_fullpath: Users/How To/
+toc_mdlink: azs-how-add-managed-disks-portal.md
+---
+
+# How to add a managed disk to a virtual machine using UKCloud Azure Stack portal
+
+## Overview
+
+UKCloud for Azure Stack supports the use of managed disks on virtual machines. Managed disks can be used as both OS disks and a data disk.
```

Comment:
Suggested change:
UKCloud for Microsoft Azure supports the use of managed disks on virtual machines (VMs). You can use managed disks as both OS disks and data disks.